packages/core/permissions/src/__tests__/permissions.engine.test.ts

```diff
@@ -1,8 +1,14 @@
-'use strict';
-
-const _ = require('lodash');
-const { subject } = require('@casl/ability');
-const permissions = require('..');
+import _ from 'lodash';
+import { subject } from '@casl/ability';
+import { providerFactory } from '@strapi/utils';
+import permissions from '..';
```

Comment:
"This module is declared with 'export =', and can only be used with a default import when using the 'esModuleInterop' flag"

Reply:
🤔  ok didn't get any error there. will check 👍

Reply:
Might be my vscode ts server acting weird again, just wanted to report it in case it's actually smth valid :eyes:

Reply:
Yup probably because we actually have esModuleInterop set to true :)